crates/ruff_linter/src/message/sarif.rs

```diff
@@ -153,6 +153,58 @@ struct SarifResult<'a> {
     start_column: OneIndexed,
     end_line: OneIndexed,
     end_column: OneIndexed,
+    fixes: Vec<SarifFix>,
+}
+
+#[derive(Debug, Serialize)]
+struct SarifFix {
+    description: RuleDescription,
+    #[serde(rename = "artifactChanges")]
+    artifact_changes: Vec<SarifArtifactChange>,
+}
+
+#[derive(Debug, Serialize)]
+
+struct RuleDescription {
+    text: Option<String>,
+}
+
+#[derive(Debug, Serialize)]
+struct SarifArtifactChange {
+    #[serde(rename = "artifactLocation")]
+    uri: SarifArtifactLocation,
```

Comment:
I'd prefer if the fields use the same names as in the Sarif specification. It makes it easier to match the fields to their output.

```suggestion
    #[serde(rename = "artifactLocation")]
    artifactLocation: SarifArtifactLocation,
```